changelog/11226.txt

```diff
@@ -0,0 +1,3 @@
+```changelog:enhancement
```

Comment:
I think this needs to be "release-note", although I am getting in here too late for that to be really useful.